ha/core/event_analyzer/watcher/watcher.py

```diff
@@ -68,36 +67,44 @@ def _validate(self) -> None:
         if not isinstance(self.subscriber, Subscriber):
             raise InvalidSubscriber(f"Invalid subscriber {self.subscriber}")
 
-    def run(self):
+    def process_message(self, message: str) -> None:
         """
-        Overloaded of Thread.
+        Callback function to get message.
```

Comment:
*to process message

Reply:
i will address it after rebase with
https://github.com/Seagate/cortx-ha/pull/542

Reply:
after message bus code it is not needed